site/docs/v1/tech/apis/identity-providers/_samlv2-request-body.adoc

```diff
@@ -70,9 +70,29 @@ The specified Lambda Id must be of type `SAMLv2Reconcile`.
 
 include::_identity-provider-linking-strategy-request-parameter.adoc[]
 
+[field]#identityProvider.loginHintConfiguration.enabled# [type]#[boolean]# [optional]#Optional# [default]#Defaults to `false`# [since]#Available since 1.43.0#::
+When enabled and HTTP-Redirect binding are used, a login hint will be passed (when available) as a query parameter to the IdP using the configured parameter name set by the [field]#identityProvider.loginHintConfiguration.parameterName#.
+
+[field]#identityProvider.loginHintConfiguration.parameterName# [type]#[String]#::
+The name of the query parameter to use when sending a login hint to the IdP. This is only used when a login hint is available and [field]#identityProvider.loginHintConfiguration.enabled# is set to `true`.
+
 [field]#identityProvider.name# [type]#[String]# [required]#Required#::
 The name of this SAML v2 identity provider. This is only used for display purposes. Must be unique across all identity providers.
 
+ifndef::samlv2_idp_initiated[]
+[field]#identityProvider.nameIdFormat# [type]#[String]# [required]#Required# [default]#Defaults to `urn:oasis:names:tc:SAML:2.0:nameid-format:persistent`#::
+The *NAME-ID*entifier property is used to facilitate communication about a user between a Service Provider (SP) and Identity Provider (IdP). The SP can specify the preferred format in the AuthN request regarding a user. The identity Provider will attempt to honor this format in the AuthN response.
++
+- `urn:oasis:names:tc:SAML:2.0:nameid-format:persistent` -
+FusionAuth will expect the user id returned will not change and is unique.
++
+- `urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress` -
+FusionAuth will expect an email address returned for the `NameId`. When setting this option, ensure email addresses within the external identity provider are globally unique and unchanging. If the external system does not have unique email addresses, identify a unique Id from the external IdP using the the `identityProvider.uniqueIdClaim` field.
++
+- `urn:oasis:names:tc:SAML:1.1:nameid-format:other` -
+Allows customization of the `nameIdFormat` to be any `NameId` format value.
```

Comment:
- If we are going to document the options, we would need to document more than this. 
- `urn:oasis:names:tc:SAML:1.1:nameid-format:other` is not a format, we never use this.
- This could be any valid format, so I don't know if we want to document all possible options. We could document a few common ones and indicate it could be any valid NameId format you want to send to the IdP.

Reply:
ah, I was going based on the admin UI  🙂 
<img width="1228" alt="image" src="https://user-images.githubusercontent.com/16090626/215607232-b5233cef-9491-4508-935d-05521b19a54a.png">